loleaflet/html/loleaflet.html.m4

```diff
@@ -330,5 +330,5 @@ m4_ifelse(MOBILEAPP,[true],
   [    <script src="%SERVICE_ROOT%/loleaflet/%VERSION%/bundle.js" defer></script>
   ])
 )m4_dnl
-    <!--%BRANDING_JS%--> <!-- logo onclick handler -->
+    <!--%BRANDING_JS%--> <!-- logo onclick handler --> <!--%CSS_VARS%-->
```

Comment:
Let's put the CSS_VARS on a new line :-)

Reply:
done